src/EventHub/EventHub/ChangeLog.md

```diff
@@ -18,6 +18,10 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+## Version 5.3.0
```

Comment:
Please remove this line. The version will be bumped up automatically during the release process.

Reply:
done